components/StoreCard.js

```diff
@@ -1,8 +1,8 @@
 import { FontAwesome, MaterialCommunityIcons } from '@expo/vector-icons';
 import React from 'react';
 import { TouchableOpacity, View } from 'react-native';
-
-import { H3, InLineContainer, Subtitle, Title } from '../styles/shared';
+import {Subhead, Body, Title,Caption} from '../components/BaseComponents';
+import { H3, InLineContainer, Subtitle } from '../styles/shared';
```

Comment:
Let's get rid of any of these that are unused now - linter should have complained